e2e/__tests__/deprecatedCliOptions.test.ts

```diff
@@ -1,25 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-import * as path from 'path';
-import runJest from '../runJest';
-
-const dir = path.resolve(__dirname, '../deprecated-cli-options');
-
-it('Prints deprecation warnings for CLI flags', () => {
```

Comment:
Having a test for this is useful, but we don't have any other deprecations?

Reply:
Yea, was wondering if we should keep it empty and leave a comment, "in case of deprecations, test here". You think it makes sense?

Reply:
not sure... is that easier than just digging up this PR again?

Reply:
Nevertheless, it would be weird to have a "placeholder" test, just in case. I'll edit the OP and adjust the title so it's easier to find when we need it